reflex/.templates/web/utils/state.js

```diff
@@ -213,6 +213,10 @@ export const applyEvent = async (event, socket) => {
     return false;
   }
 
+   // Only proceed if the socket is up, otherwise we throw the event into the void
+   if (!socket) {
```

Comment:
Delaying/moving the socket check here(as well as in `applyEvent` and `queueEvent`) to allow events that do not necessarily use websockets(think server side events like `rx.console_log`, `rx.window_alert`, etc) but use the event queue to execute.